src/index.ts

```diff
@@ -8,19 +8,30 @@ async function run() {
 
   const token = core.getInput('github-token');
   const github: GithubApi = new GithubApi(token);
-  const content: string[] = await github.getIssueContent();
+  const content: IIssueData = await github.getIssueContent();
+  const includedLabels: string[] | undefined = core.getInput('included-labels', { required: false }).replace(/\[|\]/gi, '').split('|');
+  const excludedLabels: string[] | undefined = core.getInput('excluded-labels', { required: false }).replace(/\[|\]/gi, '').split('|');
+
   const issue: Issue = new Issue(content);
-  const winningAreaData: IParameter = issue.getWinningAreaData(issue.determineArea())
+  const winningAreaData: IParameter = issue.getWinningAreaData(issue.determineArea());
+
+  if (includedLabels[0] || excludedLabels[0]) {
+    if (!issue.verifyIssueLabels(includedLabels, excludedLabels)) {
+      console.log("Issue failed label validation. Exiting successfully");
```

Comment:
The correct way to log in a github action is to use the `core.log` functions and the like.

Reply:
done!